src/pybindings/pycontroller/ControllerBindings/PyChip_ChipError.cpp

```diff
@@ -0,0 +1,63 @@
+#include <core/CHIPError.h>
+#include <sstream> // __str__
+#include <support/TypeTraits.h>
+
+#include <functional>
+#include <pybind11/pybind11.h>
+#include <string>
+
+#ifndef BINDER_PYBIND11_TYPE_CASTER
+#define BINDER_PYBIND11_TYPE_CASTER
+PYBIND11_DECLARE_HOLDER_TYPE(T, std::shared_ptr<T>)
+PYBIND11_DECLARE_HOLDER_TYPE(T, T *)
+PYBIND11_MAKE_OPAQUE(std::shared_ptr<void>)
+#endif
+
+void bind_PyChip_ChipError(std::function<pybind11::module &(std::string const & namespace_)> & M)
+{
+    { // chip::ChipError file:core/CHIPError.h line:50
```

Comment:
remove this comment: line may become obsolete in time if chiperror moves. same below.

I am working here on the assumption that these are maintained manually. If these are generated files, could we make generation part of the build process instead or does that require too much 3rd party?

Reply:
I will remove the comments. This is a semi-generated file, but will require compiling binder and manuals intervention as we do not want all SDK apis bound.